components/ChipTextField/examples/MDCChipTextFieldScrollView.m

```diff
@@ -0,0 +1,200 @@
+
+  // Set fixed values for this scroll view.
+  [self setShowsVerticalScrollIndicator:NO];
+  [self setShowsHorizontalScrollIndicator:NO];
+  self.bounces = NO;
+  self.scrollEnabled = NO;
+}
+
+- (void)appendChipView:(MDCChipView *)chipView {
+  self.trailingConstraint.active = NO;
+  [self.contentView addSubview:chipView];
+  [chipView setContentHuggingPriority:UILayoutPriorityRequired
+                              forAxis:UILayoutConstraintAxisHorizontal];
+  [chipView setContentCompressionResistancePriority:UILayoutPriorityRequired
+                                            forAxis:UILayoutConstraintAxisHorizontal];
+  NSLayoutConstraint *chipViewConstraintTop = [NSLayoutConstraint constraintWithItem:self.contentView
+                                                                           attribute:NSLayoutAttributeTop
+                                                                           relatedBy:NSLayoutRelationLessThanOrEqual
+                                                                              toItem:chipView
+                                                                           attribute:NSLayoutAttributeTop
+                                                                          multiplier:1
+                                                                            constant:0];
+  NSLayoutConstraint *chipViewConstraintBottom = [NSLayoutConstraint constraintWithItem:self.contentView
+                                                                              attribute:NSLayoutAttributeBottom
+                                                                              relatedBy:NSLayoutRelationGreaterThanOrEqual
+                                                                                 toItem:chipView
+                                                                              attribute:NSLayoutAttributeBottom
+                                                                             multiplier:1
+                                                                               constant:0];
+  NSLayoutConstraint *chipViewConstraintCenterY = [NSLayoutConstraint constraintWithItem:self.contentView
+                                                                              attribute:NSLayoutAttributeCenterY
+                                                                              relatedBy:NSLayoutRelationEqual
+                                                                                 toItem:chipView
+                                                                              attribute:NSLayoutAttributeCenterY
+                                                                             multiplier:1
+                                                                               constant:0];
+  
+  NSLayoutConstraint *chipViewConstraintLeading = nil;
+  NSInteger numberOfChips = [self.dataSource numberOfChipsOnScrollView:self];
+  if (numberOfChips == 0) {
```

Comment:
I think it would not be a correct assumption once we support inserting chip. But for now I think it is okay? I can add a comment here if that would make it clear.

Reply:
Yes. I think adding a comment would make it clear.